dotCMS/src/main/java/com/dotmarketing/filters/LoginRequiredFilter.java

```diff
@@ -39,6 +41,9 @@ private void addDefaultInterceptors(final FilterConfig config) {
 
 
         delegate.add(new DefaultBackEndLoginRequiredWebInterceptor());
```

Comment:
I wonder if we were not going to remove the DefaultBackEndLoginRequiredWebInterceptor